cluster_management/src/main/java/com/pinterest/rocksplicator/ClientShardMapAgent.java

```diff
@@ -136,7 +136,7 @@ private static CommandLine processCommandLineArgs(String[] cliArgs) throws Parse
   public static void main(String[] args) throws Exception {
     org.apache.log4j.Logger.getRootLogger().setLevel(Level.INFO);
     BasicConfigurator.configure(new ConsoleAppender(
-        new PatternLayout("%d{HH:mm:ss.SSS} [%t] %-5p %30.30c - %m%n")
+        new PatternLayout("%d{dd MMM yyyy HH:mm:ss.SSS} [%t] %-5p %30.30c - %m%n")
```

Comment:
could you add a sample log in the PR description?

Reply:
see https://github.com/pinterest/rocksplicator/pull/589, it would be the same, but I can try to see if we have test stage setup for spectators, etc

Reply:
tested a private build and updated the summary fro this.